--- pjsip/include/pjsua2/media.hpp.orig	2024-01-01 00:00:00.000000000 +0000
+++ pjsip/include/pjsua2/media.hpp	2024-01-01 00:00:00.000000000 +0000
@@ -2925,6 +2925,15 @@
     pj_status_t             status;     /**< The status code.       */
 };

+/**
+ * This structure describes the rtcp fb event.
+ */
+struct RtcpFbEvent
+{
+    pjmedia_rtcp_fb_type    fbType;
+    bool                    isParamLengthZero;
+};
+
 /**
  * Media event data.
  */
@@ -2939,6 +2948,11 @@
      */
     AudDevErrorEvent        audDevError;

+    /**
+     * Rtcp FB event data.
+     */
+    RtcpFbEvent             rtcpFb;
+
     /**
      * Pointer to storage to user event data, if it's outside
      * this struct
