--- pjsip/include/pjsua2/media.hpp.orig	2024-01-01 00:00:00.000000000 +0000
+++ pjsip/include/pjsua2/media.hpp	2024-01-01 00:00:00.000000000 +0000
@@ -2680,6 +2680,15 @@
     pj_status_t             status;     /**< The status code.       */
 };

+/**
+ * This structure describes the rtcp fb event.
+ */
+struct RtcpFbEvent
+{
+    pjmedia_rtcp_fb_type    fbType;
+    bool                    isParamLengthZero;
+};
+
 /**
  * Media event data.
  */
@@ -2694,6 +2703,11 @@
      */
     AudDevErrorEvent        audDevError;

+    /**
+     * Rtcp FB event data.
+     */
+    RtcpFbEvent             rtcpFb;
+
     /**
      * Pointer to storage to user event data, if it's outside
      * this struct
--- pjsip/src/pjsua2/media.cpp.orig	2024-01-01 00:00:00.000000000 +0000
+++ pjsip/src/pjsua2/media.cpp	2024-01-01 00:00:00.000000000 +0000
@@ -2034,6 +2034,9 @@
         data.audDevError.dir = ev.data.aud_dev_err.dir;
         data.audDevError.id = ev.data.aud_dev_err.id;
         data.audDevError.status = ev.data.aud_dev_err.status;
+    } else if (type == PJMEDIA_EVENT_RX_RTCP_FB) {
+        data.rtcpFb.fbType = ev.data.rx_rtcp_fb.cap.type;
+        data.rtcpFb.isParamLengthZero = ev.data.rx_rtcp_fb.cap.param.slen == 0;
     }
     pjMediaEvent = (void *)&ev;
 }
